app/build.gradle

```diff
@@ -105,14 +105,14 @@ dependencies {
 	}
 	implementation 'com.google.android.material:material:1.0.0'
 	implementation 'androidx.cardview:cardview:1.0.0'
-	implementation 'androidx.recyclerview:recyclerview:1.0.0'
+	implementation 'androidx.recyclerview:recyclerview:1.1.0'
 	implementation 'androidx.legacy:legacy-preference-v14:1.0.0'
 	implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
 
 	implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0'
 	implementation 'androidx.lifecycle:lifecycle-common-java8:2.1.0'
-	implementation 'androidx.room:room-runtime:2.2.0'
-	kapt 'androidx.room:room-compiler:2.2.0'
+	implementation 'androidx.room:room-runtime:2.2.2'
+	kapt 'androidx.room:room-compiler:2.2.2'
```

Comment:
Normally, I update non-test libs only short after a release, but not short before to not introduce new bugs. PTE is an exception to that rule.

Reply:
I've changed it now accordingly to update only PTE and test libraries.

Reply:
I've changed it now accordingly to update only PTE and test libraries.